strsim_func/src/lib.rs

```diff
@@ -0,0 +1,74 @@
+use strsim::damerau_levenshtein;
+use strsim::jaro;
+use strsim::jaro_winkler;
+use strsim::hamming;
+use strsim::normalized_damerau_levenshtein;
+use strsim::normalized_levenshtein;
+use strsim::levenshtein;
+use gandiva_rust_udf_macro::udf;
+
+#[udf]
+pub fn jaro_similarity(a: &str, b: &str) -> f64{
+   return jaro(a,b);
+}
+
+#[udf]
+pub fn jaro_winkler_similarity(a: &str, b: &str)->f64{
+    return jaro_winkler(a, b);
+}
+
+#[udf]
+pub fn damerau_levenshtein_distance(a: &str, b: &str) -> usize{
+    return damerau_levenshtein(a, b);
+}
+
+#[udf]
+pub fn levenshtein_distance(a: &str, b: &str) -> usize{
+    return levenshtein(a, b);
+}
```

Comment:
In gdv there's already one levenshtein c++ udf. Not sure they works same.
<img width="696" alt="image" src="https://github.com/yanhuangdata/gandiva_rust_udf/assets/59006258/9e54e3a9-455a-4f4b-a92c-3d0493eaba9b">

Reply:
we may have to remove the `levenshtein_distance` to avoid confusion